### PR TITLE
Release 1.0.8

### PR DESCRIPTION
**Pull request by trobinson-r7**
### Description
Added additional debug logging for improved request + response info

### Motivation and Context
Customer is having an issue here https://issues.corp.rapid7.com/browse/LSI-1403 but we don't have sufficient logging to what where the issue with the request is

### How Has This Been Tested?
https://wiki.corp.rapid7.com/display/EXT/ADO+v1.0.8+test+plan

### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist:

- [x] I have tested the changes locally.

### CODE REVIEW

File: tasks/InsightAppSec/helpers/insightAppSecApi.ts [hunk 1]
```diff
@@ -27,6 +29,13 @@ export default class InsightAppSecApi
             transformResponse: [data => data]
         });
 
+        this.axiosInst.interceptors.request.use(request => {
+            if(debugMode){
+                console.log('##[debug]Request made: ', JSON.stringify(request, null))
```
Comment (ckearney-r7) [inline tasks/InsightAppSec/helpers/insightAppSecApi.ts:34]: console.debug doesn't show?

File: tasks/InsightAppSec/task.ts [hunk 2]
```diff
@@ -17,6 +17,13 @@ async function run() {
         var hasScanGating = tl.getBoolInput("hasScanGating");
         var generateFindingsReport = tl.getBoolInput("generateFindingsReport");
 
+        var debugModeStr = tl.getVariable("system.debug");
```
Comment (ckearney-r7) [inline tasks/InsightAppSec/task.ts:20]: Is this available in both build and release pipelines?

**Comment by ckearney-r7** [comment]
Can we update the extension.spec.yaml file too with the latest version and maybe add an entry in the help.md?

File: tasks/InsightAppSec/helpers/insightAppSecApi.ts (hunk 1 above)
Comment (trobinson-r7) [reply tasks/InsightAppSec/helpers/insightAppSecApi.ts:34]: No, didn't work for me

File: tasks/InsightAppSec/task.ts (hunk 2 above)
Comment (trobinson-r7) [reply tasks/InsightAppSec/task.ts:20]: Yes it is, need to add system.debug=true pipeline variable in the config

**Review by ckearney-r7 — APPROVED** [review]

**Pull request merged**